tests/snmp_test.py

```diff
@@ -30,6 +30,13 @@ async def test_getnext(self, snmp_client):
         assert isinstance(response.oid, OID)
         assert isinstance(response.value, int)
 
+    @pytest.mark.asyncio
+    async def test_getnext2_should_return_symbolic_identifiers(self, snmp_client):
+        response = await snmp_client.getnext2(("IF-MIB", "ifName", "1"), ("IF-MIB", "ifAlias", "1"))
+        assert len(list(response)) == 2
+        assert any(k == Identifier("IF-MIB", "ifName", OID(".2")) for k, v in response)
+        assert any(k == Identifier("IF-MIB", "ifAlias", OID(".2")) for k, v in response)
```

Comment:
Was a bit unsure what k and v was reading this. I assumed key and value, implying a dictionary, but that aint right.
Might be clearer to write like this:
```suggestion
        assert any(identifier == Identifier("IF-MIB", "ifName", OID(".2")) for identifier, _ in response)
        assert any(identifier == Identifier("IF-MIB", "ifAlias", OID(".2")) for identifier, _ in response)
```